From b96ffcdaef3532eb4e07d53d92a29d4a89d3fd67 Mon Sep 17 00:00:00 2001
From: "Wang, ArvinX" <arvinx.wang@intel.com>
Date: Tue, 23 Oct 2018 10:29:26 +0800
Subject: [PATCH] Fix LocalMediaPlayer was killed by changing the display size

LocalMediaPlayer was killed when the density is changed. The AMS
always killed the background service and the target sdk version < 24.

Change-Id: I18403445067f847e606c14d41e642f0feccb9406
Tracked-On:
Signed-off-by: Wang, ArvinX <arvinx.wang@intel.com>
---
 AndroidManifest.xml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/AndroidManifest.xml b/AndroidManifest.xml
index 22ece79..ed004e4 100644
--- a/AndroidManifest.xml
+++ b/AndroidManifest.xml
@@ -20,7 +20,7 @@

     <uses-sdk
         android:minSdkVersion="23"
-        android:targetSdkVersion="23" />
+        android:targetSdkVersion="24" />

     <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" />

--
2.21.0
